pubspec.yaml

```diff
@@ -18,9 +18,9 @@ dependencies:
   w_common: ^1.0.0
 dev_dependencies:
   coverage: ^0.7.9
-  dart_dev: ^1.5.1
-  dart_style: ^0.2.11+1
-  dartdoc: ^0.9.7+6
+  dart_dev: ^1.7.7
+  dart_style: '>=0.2.4 <2.0.0'
+  dartdoc: '>=0.9.0 <0.13.0'
```

Comment:
when dependencies are pre 1.0, the minor value increment constitutes a breaking change, so we upper bound at the next breaking version